src/include/txn.h

```diff
@@ -170,6 +170,27 @@ struct __wt_txn_global {
     volatile uint64_t metadata_pinned; /* Oldest ID for metadata */
 
     WT_TXN_SHARED *txn_shared_list; /* Per-session shared transaction states */
+
+#define WT_TXN_GLOBAL_FOREACH_SESSION_STATE(state, conn)                                           \
```

Comment:
This name is a bit clunky but I didn't like `WT_TXN_GLOBAL_FOREACH_TXN_SHARED`. Open to any suggestions

Reply:
WT_TXN_SHARED_FOREACH_BEGIN?

Reply:
Renamed in [9870153](https://github.com/wiredtiger/wiredtiger/pull/9296/commits/987015300d15a748e41cd37eb82f30ce002adac3)